src/java.base/share/classes/java/util/concurrent/ThreadLocalRandom.java

```diff
@@ -506,6 +506,8 @@ public float nextFloat() {
      * {@inheritDoc}
      * @throws IllegalArgumentException {@inheritDoc}
      * @implNote {@inheritDoc}
+     *
+     * @since 17
      */
     @Override
     public float nextFloat(float bound) {
```

Comment:
These 2 `nextFloat(...)` methods which take parameters were introduced in Java 17 https://bugs.openjdk.org/browse/JDK-8248862, whereas the no-arg `nextFloat()` method existed prior to that. So adding `@since 17` to these 2 methods looks correct to me.